mdcli/validate.go

```diff
@@ -26,21 +26,14 @@ func Validate(opts *CommandOptions) (int, error) {
 	)
 
 	valErr, err := mdProcessor.Validate(cli)
+
 	if err != nil {
-		return 0, err
-	}
-	if valErr == nil {
-		fmt.Fprintf(opts.Stdout, "%s: OK\n", configPath)
-		return 0, nil
+		if valErr != nil {
+			fmt.Fprintf(opts.Stderr, "Error: %s\nReason: %s\n", valErr.Error, valErr.Message)
+		}
+		return 1, err
```

Comment:
okay, this is better I think, one more nit.  If we have `valErr`, I dont think we want to return `err` as well.  So something like this:
```
if err != nil {
		if valErr != nil {
			fmt.Fprintf(opts.Stderr, "Error: %s\nReason: %s\n", valErr.Error, valErr.Message)
			return 1, nil
		}
		return 1, err
}
```
since we are "handling" the error by printing out the message, dont think we need to return it (which will likely cause it to get printed again from the `main`.

Reply:
Thanks for the feedback!